### PR TITLE
Internationalisation

### PR DESCRIPTION
**Pull request by steveWinter**
This PR continues on from the previous one which began the process by moving (most of) the onscreen text into a French translation file.

Within this PR we add en_NZ translations as well as allowing their use within the codebase, translating URIs, adding NZ postcodes etc

The one remaining thing which is needed is the ability to use translations in the React components of the site.

### CODE REVIEW

File: templates/content/en_NZ/legalities.html.twig [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            <h3 class="text-center font-weight-bold mb-4">
+    Legal</h3>
+
+<div class="process-details">
+    <h5 class="font-weight-bold">
+        Publisher</h5>
+
+    <ul>
+        <li>
+            Titouan Galopin : <a href="mailto:titouan@enpremiereligne.fr">titouan@enpremiereligne.fr</a>
```
Comment (tgalopin) [inline templates/content/en_NZ/legalities.html.twig:15]: Pretty sure we are going to need to update this, but let's do it afterwards :)

File: templates/content/en_NZ/who.html.twig [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
    </ul>

    <h5 class="font-weight-bold mt-5">
        Point de contact de l'équipe entière :
    </h5>

    <p>
        <a href="mailto:team@enpremiereligne.fr">team@enpremiereligne.fr</a>
    </p>
</div>
```
Comment (tgalopin) [inline templates/content/en_NZ/who.html.twig:6]: There seems to be something weird here in Github display, not sure why

File: templates/emails/match-footer.html.twig [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    {% trans %}email.good-luck{% endtrans %}
+</p>
+
+<p>
+    {% trans %}email.signoff{% endtrans %}
+</p>
+
+<p>
+    {% trans %}email.site-helped{% endtrans %}
+</p>
```
Comment (tgalopin) [inline templates/emails/match-footer.html.twig:19]: Missing EOF

File: templates/emails/match_groceries.html.twig [hunk 4]
```diff
@@ -1,186 +1,36 @@
...
-
-<p>
-    L'équipe d'En Première Ligne
-</p>
-
-<p>
-    PS : si notre plateforme vous a aidé(e), n'hésitez pas à nous le dire à
-    <a href="mailto:team@enpremiereligne.fr">team@enpremiereligne.fr</a>. Nous sommes à votre disposition !
-</p>
+{% include 'emails/match-footer.html.twig' %}
```
Comment (tgalopin) [inline templates/emails/match_groceries.html.twig:36]: Missing EOF

File: templates/emails/match_guidelines.html.twig [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    <li>
+        {% trans %}email.no-shaking-hands{% endtrans %}
+    </li>
+    <li>
+        {% trans %}email.use-tissues{% endtrans %}
+    </li>
+    <li>
+        {% trans %}email.avoid-gatherings{% endtrans %}
+    </li>
+</ul>
```
Comment (tgalopin) [inline templates/emails/match_guidelines.html.twig:51]: Missing EOF

File: templates/content/en_NZ/legalities.html.twig (hunk 1 above)
Comment (steveWinter) [reply templates/content/en_NZ/legalities.html.twig:15]: Hmm - I thought that Emmanuel had updated all of those. I know that he's still working on getting email set up for the NZ domain, so maybe that's why it's not been done.

File: templates/emails/match_guidelines.html.twig (hunk 5 above)
Comment (steveWinter) [reply templates/emails/match_guidelines.html.twig]: Updated

**Pull request merged**